ro/cs-100/solve-oversized-payload-issue (#111)
MindscapeHQ/raygun4py: CHANGELOG.md

## 5.0.0 (13/02/24):
Breaking changes:
  - Support for Python 2.7 has been dropped

Bug Fixes:
  - Removes largest local/global variables in crash payload to ensure payload size limit is not breached (optional)

>     - We now provide `enforce_payload_size_limit` and `log_payload_size_limit_breaches` as configuration options, these default both being enabled

Reviewer: ```suggestion
    - We now provide `enforce_payload_size_limit` and `log_payload_size_limit_breaches` as configuration options, both enabled by default
```

>     - These control whether the provider will attempt to remove variables if there is an oversized payload and log what varaible were removed

Reviewer: ```suggestion
    - These control whether the provider will attempt to remove variables if there is an oversized payload and log which variables were removed
```
